vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -40,7 +40,6 @@ type VerticalPodAutoscalerList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:storageversion
 // +kubebuilder:resource:shortName=vpa
-// +kubebuilder:subresource:status
```

Comment:
Please don't remove this.

Reply:
It looks like a 'Revert "Add subresource status for vpa"' commit got rolled into a squash.    I'll try a revert-revert and then if that works, squash that.